bears/shell/BashateBear.py

```diff
@@ -0,0 +1,32 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='bashate',
+        output_format='regex',
+        output_regex=r'\[.*\] (?P<origin>E\d+): '
+                     r'(?P<message>.+):(?:.*\s.*): '
+                     r'L(?P<line>\d+)')
+class BashateBear:
+    """
+    Bashate is a style-checker for bash scripts.
+    The name is derived from pep8, a Python lint-type tool.
```

Comment:
Not sure this sentence is important, you want to keep it? :)
(In case you want, add another blank line above so it counts as "detailed explanation")

Reply:
no 👍